client/src/components/AdminPanel/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import './Sidebar.scss';
+import {Role} from '../../../utils/authentication-util';
+import {NavLink} from 'react-router-dom';
```

Comment:
Minor: Should be `{ NavLink }` (spacing)

Reply:
My IntelliJ settings got all messed up when I was changing them for 344-rest 😢